community-handbook/community-report.md

```diff
@@ -0,0 +1,33 @@
+# Community Report
+
+The CHAOSS Community Report is a free service that the CHAOSS community provides to open source communities.
+
+## Goals
+
+* Create awareness for the CHAOSS project
+* Showcase the CHAOSS metrics and software
+* Get more communities interested in CHAOSS metrics and software
+* Grow the CHAOSS community
+
+## Roles
+
+The Communit Report processes have the following roles:
+
+* Requester -- Someone asking for a Community Report
+* Coordinator -- The CHAOSS contact person coordianting the creation and distribution of Community Reports
+
+## Requesting a Community Report
+
+Anyone can fill out a form on the CHAOSS.community website. The form submits to a Linux Foundation system. 
+
+## Generating a Community Report
+
+When the CHAOSS community receives a request for a report, the following steps will be taken:
+
+* TBD by TBD
```

Comment:
?? :)

Reply:
I might suggest this here:

1. Coordinator notifies points of contact from Augur and Grimoire Lab a request has been submitted.
2. Augur and Grimoire Lab contacts generate visual reports for the requested repos.
3. Augur and Grimoire Lab contacts send visuals to Coordinator.
4. Coordinator compiles visuals into a distributable PDF document and sends back to the requester.

Reply:
I might suggest this here: 
* Coordinator notifies points of contact from Augur and GrimoireLab a request has been submitted.

Reply:
Does this have to be manual?  The coordinator and point of contact from Augur and GrimoireLab should be notified through an autogenerated email once the form is submitted. A coordinator can followup for getting desired visualizations for generation of pdf.

Reply:
@vinodkahuja I think they do get those emails; this was more of a failsafe so nothing falls through the cracks. Maybe we don't need that though. 🤔  It definitely would streamline the process a little!